bot/exts/utils/extensions.py

```diff
@@ -34,6 +34,7 @@ class Extensions(commands.Cog):
 
     def __init__(self, bot: Bot):
         self.bot = bot
+        self.action_in_progress: bool = False
```

Comment:
I feel like it is unnecessary to add an annotation here.

```suggestion
        self.action_in_progress = False
```

Reply:
Removed, and I rebased onto main.